swig/cpp/src/Connection.h

```diff
@@ -31,10 +31,10 @@ extern "C" {
 #include "sysrepo.h"
 }
 
-class Connection:public Throw_Exception
+class Connection
 {
 public:
-    Connection(const char *app_name, const sr_conn_options_t opts = SESS_DEFAULT);
+    Connection(const char *app_name, const sr_conn_options_t opts = 0);
```

Comment:
What is the motivation behind this change? As an API user, I have no idea what ``0`` means. Given that the code already requires C++11, have you considered going with scoped enums (``enum class``)? (And "something" for flags, yeah...)